language_service/src/lib.rs

```diff
@@ -221,6 +222,17 @@ impl LanguageService {
         )
     }
 
+    /// LSP: textDocument/format
+    #[must_use]
+    pub fn get_format_changes(&self, uri: &str) -> Vec<TextEdit> {
+        self.document_op(
```

Comment:
Does this have the correct encoding offsets applied?

Reply:
The offsets encoding is fine as this function is of the same form as the other LS feature functions in this file. I believe @minestarks recently (within the last 2 months) made changes to this infrastructure regarding encoding, so she might know more.

Reply:
Yeah the offset/position mappings should all fall into place.